src/main/scala/it/pagopa/interop/agreementprocess/error/AgreementProcessErrors.scala

```diff
@@ -91,4 +91,10 @@ object AgreementProcessErrors {
   final case class TenantIdNotFound(tenantId: UUID)
       extends ComponentError("0020", s"Tenant ${tenantId.toString} not found")
 
+  final case class NotValidEServiceDescriptorId(descriptorId: UUID)
```

Comment:
This way the specific error reason is clear by just reading the name
```suggestion
  final case class NotLatestEServiceDescriptor(descriptorId: UUID)
```